### ISSUE TITLE
use constant value instead of calculation and change order of uint8, uint256 for packing 

### ISSUE THREAD

**Issue by FlokiBB** [DeDogmaDao/dt-contracts#27]
https://github.com/DeDogmaDao/dt-contracts/blob/d3e73969d0502b760959991224343c179de762d2/src/contracts/NFT.sol#L69

**Comment by dante6363** [comment]
just gas optimizing? cause it is already constant

**Comment by FlokiBB** [comment]
yes. actually, I just want to get rid of multiply in transaction process time to reduce gas consumption
